travis-setup.md

```diff
@@ -0,0 +1,88 @@
+# Building and Deploying with Travis
+
+### Setup Travis web
+ 1) To get started with Travis, follow the first 3 steps in the following tutorial:
+    - https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci
+    - For step 3, make sure you include activating your fork of this repository
+
+### Create Firebase account
+ 1) Navigate to https://console.firebase.google.com
+ 2) Sign in/make an account
+ 3) Create a new project:
+    - For project name, use `{your username}-tba-dev`
+    - Leave the other settings to the defaults, unless you know what you're doing
+
+    ![](https://i.imgur.com/tk4kWtn.png)
+
+ 4) On the left, navigate to Develop -> Storage
+    - After clicking through a few confirmations, you should be on a screen like this:
+
+    ![](https://i.imgur.com/tpFcxrU.png)
+
+ 5) Update the `firebase_bucket` in `.travis.yml`:
```

Comment:
Ah gotcha - can we set this as an env variable maybe?

Reply:
Changed in `7d83030ed2c71dbcbb5c3775dd5c649ceb57b5a9`